Voice/ToolkitAPI/README.md

```diff
@@ -0,0 +1,7 @@
+# Service Cloud Voice Toolkit API
+
+This Lightning component illustrates how to use the Service Cloud Voice Toolkit API.
+
+To learn about Voice, see [Service Cloud Voice in Salesforce Help](https://help.salesforce.com/articleView?id=voice_about.htm&type=5). Also refer to the [Voice Developer Guide](https://developer.salesforce.com/docs/atlas.en-us.voice_developer_guide.meta/voice_developer_guide/voice_intro.htm).
+
+* [Sample Component](SampleComponent/): This Aura Lightning component uses the basic functionality of the Toolkit API.
```

Comment:
"basic functionality" probably doesn't need this qualifier since the functionality shown is more or less the full capability.